geostore/logging_keys.py

```diff
@@ -0,0 +1,6 @@
+LOG_MESSAGE_LAMBDA_START = "Lambda:Start"
+LOG_MESSAGE_LAMBDA_FAILURE = "Lambda:Failure"
+LOG_MESSAGE_VALIDATION_FAILURE = "Validation:Failure"
+LOG_MESSAGE_VALIDATION_SUCCESS = "Validation:Success"
+LOG_MESSAGE_S3_BATCH_RESPONSE = "S3Batch:Response"
+LOG_MESSAGE_STEP_FUNCTION_RESPONSE = "StepFunction:Response"
```

Comment:
As elsewhere, it's probably best if these are two separate fields in the JSON rather than one, for easier search.

Reply:
Could you please provide an example?  I don't quite understand.

Reply:
is it something like.... 
log("Validation", result="Failure", foo=bar)
log("Validation", result="Success", foo=bar)